package.json

```diff
@@ -346,10 +346,10 @@
     "VisualStudioExptTeam.vscodeintellicode"
   ],
   "dependencies": {
-    "@iconify-icons/codicon": "^1.2.8",
+    "@iconify-icons/codicon": "1.2.8",
```

Comment:
Every time this lib gets updated, we have to create corresponding license audit request due to governance. Here I simply pin this version because we don't use new icons from newer versions. We can update it later on demand.